autoscheduler/frontend/src/components/LandingPage/About/About.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import {
+  Dialog, DialogContent, DialogTitle, IconButton, Link, ThemeProvider, Typography,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import { GitHub, LinkedIn } from '@material-ui/icons';
+import { Contributor, contributors, pastContributors } from '../../../types/Contributors';
+import * as styles from './About.css';
+import { whiteButtonTheme } from '../../../theme';
+
+/**
+ * Creates a row to be used in the contributor table
+ * @param contributor the Contributor to make the row for
+ */
+function getContributorRow(contributor: Contributor): JSX.Element {
+  return (
+    <div className={styles.contributorsRow} key={contributor.name}>
+      <Typography>
+        {contributor.name}
+      </Typography>
+      <Typography variant="subtitle1">
+        {contributor.position}
+      </Typography>
+      <div>
+        <div>
+          {contributor.githubURL ? (
+            <IconButton
+              className={styles.overrideIconButton}
+              onClick={(): void => { window.open(contributor.githubURL); }}
+            >
+              <GitHub />
+            </IconButton>
+          ) : null}
+          {contributor.linkedInURL ? (
+            <IconButton
+              className={styles.overrideIconButton}
+              onClick={(): void => { window.open(contributor.linkedInURL); }}
+            >
+              <LinkedIn />
+            </IconButton>
+          ) : null}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+const About: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+
+  const contributorsJSX: JSX.Element[] = contributors.map(
+    (contributor) => getContributorRow(contributor),
+  );
+  const pastContributorsJSX: JSX.Element[] = pastContributors.map(
+    (contributor) => getContributorRow(contributor),
+  );
```

Comment:
```suggestion
  const contributorsJSX: JSX.Element[] = contributors.map(getContributorRow);
  const pastContributorsJSX: JSX.Element[] = pastContributors.map(getContributorRow);
```
Since `getContributorRow` is already a function that takes a single argument you don't need to wrap it in an arrow function